clusters/kind-cluster/infra/ory/oathkeeper/rules/rules.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: oathkeeper.ory.sh/v1alpha1
+kind: Rule
+metadata:
+  name: oathkeeper-example
+  namespace: dscp
+spec:
+  match:
+    url: <.*>/<{alice,bob}>/<{inteli-api,dscp-api,dscp-identity-service}>/<.*>
```

Comment:
this is fine for now. I assume we'll define another rule in another PR to allow the swagger stuff through?

Reply:
We have two options, we can either add a second rule which allows the swagger and api-docs routes to be unauthenticated. Or we can alter the helm charts to have unauthenticated paths as we did in wasp.